install_config/configuring_nsxtsdn.adoc

```diff
@@ -0,0 +1,209 @@
+node01.example.com ansible_ssh_host=192.168.220.5 OpenShift_node_group_name='node-config-infra' OpenShift_ip=192.168.220.5
+#node02.example.com ansible_ssh_host=192.168.220.6 OpenShift_node_group_name='node-config-infra' OpenShift_ip=192.168.220.6
+node03.example.com ansible_ssh_host=192.168.220.7 OpenShift_node_group_name='node-config-compute' OpenShift_ip=192.168.220.7
+node04.example.com ansible_ssh_host=192.168.220.8 OpenShift_node_group_name='node-config-compute' OpenShift_ip=192.168.220.8
+----
+
+Check prerequisites for {product-title} with NSX-T CNI:
+----
+$ ansible-playbook -i hosts openshift-ansible/playbooks/prerequisites.yml
+----
+
+Once the above playbook has executed successfully, run the following on all nodes:
+
+Assuming NCP Container image is downloaded locally on all nodes.
+----
+$ docker load -i nsx-ncp-rhel-xxx.tar
+----
+
+Get the image name and retag it:
+----
+$ docker images
+$ docker image tag registry.local/xxxxx/nsx-ncp-rhel nsx-ncp
+----
+
+Last step is to deploy the {product-title} cluster:
+----
+$ ansible-playbook -i hosts openshift-ansible/playbooks/deploy_cluster.yml
+----
+
+Once it is complete validate that the NCP and nsx-node-agent PODs are running:
+----
+$ oc get pods -o wide -n nsx-system
+NAME                   READY     STATUS    RESTARTS   AGE       IP              NODE                                   NOMINATED NODE
+nsx-ncp-5sggt          1/1       Running   0          1h        192.168.220.8   node04.example.com     <none>
+nsx-node-agent-b8nkm   2/2       Running   0          1h        192.168.220.5   node01.example.com     <none>
+nsx-node-agent-cldks   2/2       Running   0          2h        192.168.220.8   node04.example.com     <none>
+nsx-node-agent-m2p5l   2/2       Running   28         3h        192.168.220.4   master03.example.com   <none>
+nsx-node-agent-pcfd5   2/2       Running   0          1h        192.168.220.7   node03.example.com     <none>
+nsx-node-agent-ptwnq   2/2       Running   26         3h        192.168.220.2   master01.example.com   <none>
+nsx-node-agent-xgh5q   2/2       Running   26         3h        192.168.220.3   master02.example.com   <none>
+----
+
+== Check NSX-T after {product-title} deployment
+
+Check routing. T1 routers has been created during namespace creation and linked to T0 router:
+image:images/nsxt-routing.png[NSX routing]
+
+Observe Network Traceflow and visibility, for example between 'console' and 'grafana'. +
+To help fully understand, secure and optimise communications between pods, projects, virtual machines and external services: +
+image:images/nsxt-visibility.png[NSX visibility]
+
+Check Load Balancing. NSX-T offers Load Balancer and Ingress Controller capabilities:
+image:images/nsxt-loadbalancing.png[NSX loadbalancing]
+
+For additional configuration and options please refer to the link:https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.4/rn/NSX-Container-Plugin-Release-Notes.html[VMware NSX-T v2.4 OpenShift Plug-In] documentation.
```

Comment:
@RobbieJVMW This link goes to the release notes. Is that the best place to find _configurations and options_?